guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -1,6 +1,9 @@
 [id="performing-post-upgrade-tasks_{context}"]
 = Performing post-upgrade tasks
 
+* Optional: If you are utilizing the oVirt compute resource, it's crucial to migrate your data by executing the `foreman-rake ovirt:drop` task.
```

Comment:
Suggestions to address style requirements: Attribute for oVirt, avoiding contractions ("it's"), and keep it simple by using plain imperative voice ("migrate your data" rather than "it is crucial to migrate your data").

Suggestion to improve clarity: Where do I run it? On the server, right?

```suggestion
* Optional: If you are utilizing the {oVirt} compute resource, migrate your data by executing the `foreman-rake ovirt:drop` task on your {ProjectServer}.
```